### DISCUSSION TITLE
Condition by Flows Field "Not Exists"?

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15598] [Archive]
Hi,

how to set Rules in Flows Condition: if some field IS NOT EXISTS in Trigger?

Best Regards,
Tony

**Comment by rynskiy** [comment]
I have tried this Condition, but unfortunately it's doesn't work 
```
{
    "$trigger": {
        "payload": {
            "article_type": {
                "_eq": "_nnull"
            }
        }
    }
}
```

Actually, I need to set condition IF field "article_type" DOESN'T EXIST in $trigger/payload...
![condition](https://user-images.githubusercontent.com/54798086/190592640-d6690802-2aa8-48b1-87fa-3903d0691988.PNG)

**Comment by rynskiy** [comment]
And it seems that this condition doesn't work correctly too... :( When the trigger payload does not contain the "article_type" key the condition operation does not reject...
![nempty](https://user-images.githubusercontent.com/54798086/190593927-521bde4e-3915-451a-96f9-d68fd71ceb42.PNG)

**Comment by rynskiy** [comment]
Is anybody here? ;)

**Comment by lukas-schaetzle** [comment]
This is a known bug. See [#14571](https://github.com/directus/directus/issues/14571) for the issue. For the time being, you can work around it with the script operation (see https://github.com/directus/directus/discussions/15479#discussioncomment-3593966)
